uber/templates/regform.html

```diff
@@ -97,6 +97,12 @@
         <label for="full_name" class="col-sm-2 control-label">Legal Name</label>
         <div class="col-sm-6">{{ attendee.full_name }}</div>
     </div>
+    <div class="form-group">
+        <label for="badge_type" class="col-sm-2 control-label">Badge Type</label>
+        <div class="col-sm-6">
+            {{ attendee.badge_type_label }}
+        </div>
+    </div>
```

Comment:
If memory serves, putting this here will cause it to show up on ALL forms, including:
- the prereg form
- the form to claim a group badge
- the admin form
- the confirmation form
- the badge transfer form
- the on-site registration form

It's probably okay for it to show up in most of those places, though I'm a little unclear about whether the prereg form is one of them.  In particular, there are some weird badge types like ``PSEUDO_GROUP_BADGE`` and ``PSEUDO_DEALER_BADGE`` which I'm not sure we want to display on the prereg form.